packages/replay/src/launchdarkly.ts

```diff
@@ -0,0 +1,118 @@
+import dbg from "./debug";
+import { initialize, LDClient, LDLogger, basicLogger } from "launchdarkly-node-client-sdk";
+
+const debug = dbg("replay:launchdarkly");
+
+export type UserFeatureProfile = {
+  type: "user";
+  id: string;
+};
+export type WorkspaceFeatureProfile = {
+  type: "workspace";
+  id: string;
+};
+
+export const AnonymousFeatureProfile = {
+  type: "anonymous",
+  id: "anonymous",
+};
+
+export type FeatureProfile =
+  | typeof AnonymousFeatureProfile
+  | UserFeatureProfile
+  | WorkspaceFeatureProfile;
+
+class NoOpLogger implements LDLogger {
+  error() {}
+  warn() {}
+  info() {}
+  debug() {}
+}
+
+class LaunchDarkly {
+  private client: LDClient;
+  private initialized = true;
+  private lastCall = Date.now();
```

Comment:
If I read correctly your comments then this should be updated in `variant`

Reply:
if u'd be doing that then I think it would be best to use an actual debounce in `reClose` instead of checking the time manually etc.

The current logic is also not completely clear to me and I'm unsure if it's by design or not. For example, when you initialize the client you don't actually call `reClose` (so its timer is not set up) - only `identify` and `variant` lead to `reinitialize` and thus to `reClose`.

It feels to me that it would be more predictable if this timer would be tightly coupled with `this.initialized`.